app/controllers/sign_up/completions_controller.rb

```diff
@@ -0,0 +1,24 @@
+module SignUp
+  class CompletionsController < ApplicationController
```

Comment:
We can check that a user has completed proofing if they are doing an LOA3 flow. 

If LOA1, we will only check that they are completed signed in.